test/controllers/income/estimatedPay/update/IncomeUpdatePayslipAmountControllerSpec.scala

```diff
@@ -58,20 +58,22 @@ class IncomeUpdatePayslipAmountControllerSpec
         journeyCacheService,
         mock[FormPartialRetriever],
         MockTemplateRenderer) {
-    when(journeyCacheService.mandatoryValueAsInt(Matchers.eq(UpdateIncome_IdKey))(any()))
-      .thenReturn(Future.successful(employer.id))
-    when(journeyCacheService.mandatoryValue(Matchers.eq(UpdateIncome_NameKey))(any()))
-      .thenReturn(Future.successful(employer.name))
+    when(journeyCacheService.mandatoryJourneyValueAsInt(Matchers.eq(UpdateIncome_IdKey))(any()))
+      .thenReturn(Future.successful(Right(employer.id)))
+    when(journeyCacheService.mandatoryJourneyValue(Matchers.eq(UpdateIncome_NameKey))(any()))
+      .thenReturn(Future.successful(Right(employer.name)))
   }
```

Comment:
Missing left tests

Reply:
I have wrote test for the Get call this controller taxablePayslipAmountPage (line 245) but not for the post call handleTaxablePayslipAmount (line 269). Will write this test and do the same for all other controllers in this journey.

Reply:
done